storybook/pages/StatusItemSelectorPage.qml

```diff
@@ -55,6 +64,7 @@ ColumnLayout {
 
     StatusButton {
         Layout.alignment: Qt.AlignHCenter
+
```

Comment:
empty line

Reply:
hmm, I tend to separate attached properties from the rest, isn't it ok?